tests/ui/Property-Class-UI/exact_div/expected

```diff
@@ -0,0 +1,12 @@
+RESULTS:
+Check 1: main.exact_div.1
```

Comment:
Add "\" at the end of a line to make sure the next line appears on the next line in the output.

Reply:
Thanks, will do.

Reply:
Sorry, it should have been \\, but looks like GitHub removed it. See https://github.com/model-checking/kani/blob/main/tests/expected/reach/assert/reachable_fail/expected for an example.

Reply:
Oh yeah, sure. Let me quickly update this.

Reply:
updated